config/features.yml

```diff
@@ -142,6 +142,9 @@ features:
     actor_type: user
     description: Enables users to access the claim letters page
     enable_in_development: true
+  cst_use_lighthouse:
+    actor_type: user
+    description: When enabled, claims status tool uses the Lighthouse API instead of EVSS
```

Comment:
I would go ahead and add the `enable_in_development` key here and set it to true